ui/pages/home/home.component.js

```diff
@@ -666,10 +765,25 @@ export default class Home extends PureComponent {
           {isPopup && !connectedStatusPopoverHasBeenShown
             ? this.renderPopover()
             : null}
+          {
+            ///: END:ONLY_INCLUDE_IN
+          }
           <div className="home__main-view">
             {process.env.MULTICHAIN ? null : <MenuBar />}
             <div className="home__balance-wrapper">
-              <EthOverview />
+              {
+                ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
+                <EthOverview />
```

Comment:
we can add some simple snapshot tests to verify the dom, wdyt?

Reply:
same as above 👍🏼